### DISCUSSION TITLE
Neovim nightly

### DISCUSSION THREAD

**Discussion by pseudoparenchymatous** [nix-community/kickstart-nix.nvim#32] [Q&A] [answered]
Is this not compatible to use with https://github.com/nix-community/neovim-nightly-overlay ?

**Comment by mrcjkb** [comment, marked as answer]
Yes it is. See [this config](https://github.com/pierrot-lc/nvim-nix) for example.

**Reply by pseudoparenchymatous** [reply]
Thank you
